src/components/pages/Profile/History/ComputeJobs/Details.tsx

```diff
@@ -114,12 +114,12 @@ export default function Details({
             />
           )}
           <MetaItem title="Job ID" content={<code>{job.jobId}</code>} />
-          {job.resultsDid && (
+          {/* {job.resultsDid && (
             <MetaItem
               title="Published Results DID"
               content={<code>{job.resultsDid}</code>}
             />
```

Comment:
thought(non-blocking): Comments should be removed in normal circumstances but since is rather a quick fix we can leave it as is.